generate_csv.py

```diff
@@ -50,6 +51,34 @@ def generate_csv(mirror_dir, output_csv):
     print(f"CSV generated at {output_csv}")
 
 
+def generate_updated_csv(updated_pkg, output_csv):
```

Comment:
Generally, need to also the update the global `simple/index.html` if a project has been added/removed. I think it's safest to just update it every single time: harmless.